src/emulator/cpu.ts

```diff
@@ -185,15 +191,19 @@ export class Cpu {
                 return instruction.cycles;
 
             case Operation.call: {
-                this.clock.increment(instruction.cycles);
+                const condition = this.evaluateCondition(instruction);
+                const cycles = instruction.cycles + (condition ? 3 : 0);
```

Comment:
This gives the wrong cycle count for unconditional `CALL`  unless we adopt the convention that all branch instructions have their minimum cycle count in the definition, which I would prefer.